frontend/app/styles/app.scss

```diff
@@ -0,0 +1,315 @@
+}
+.border-2{
+  border: 2px solid $light-grey;
+}
+
+.border-d-2{
+  border-bottom: 2px solid $light-grey;
+}
+a{
+  color: inherit;
+  text-decoration: none;
+  background-color: transparent;
+  &:hover{
+    color: #0056b3;
+    text-decoration: none !important;
+  }
+}
+
+.c-grey{
+  color: grey;
+}
+.bg-white{
+  background-color: white;
+}
+
+//-----------------------Top-bar and Navbar------------------------------------
+#top-bar{
+  background-color: $dark-grey;
+}
+#nav-items{
+  flex-grow: 0;
+}
+
+#page-nav{
+
+  @media screen and(min-width:576px) {
+    padding-right: 15px;
+    padding-left: 15px;
+  }
+  
+}
+
+#items{
+  padding-top:0.5em;
+  padding-bottom: 0.5em; 
+  .nav-item{
+    padding-right: 0.5rem;
+  }
+}
+.nav-text{
+  padding-top:0.5em;
+  padding-bottom: 0.5em; 
+}
+
+.nav-link{
+  color: black;
+}
+
+#top-bar{
+  .container{
+    display: flex;
+    justify-content: space-between;
+
+    .nav-link{
+      color: white;
+    }
+  }
+}
+
+.navbar-white .navbar-toggler{
+  border: 1px solid $light-grey;
+}
+
+.navbar-white .navbar-toggler-icon {
+  background-image: url("data:image/svg+xml;charset=utf8,%3Csvg viewBox='0 0 30 30' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath stroke='rgba(0, 0, 0, 0.5)' stroke-width='2' stroke-linecap='round' stroke-miterlimit='10' d='M4 7h22M4 15h22M4 23h22'/%3E%3C/svg%3E");
+}
+
+
+.input-group{
+  border: 2px solid $light-grey;
+}
+#btn-search{
+  background-color: transparent;
+}
+
+input[type="search"], input[type="password"], textarea, select { 
+  border: none;
+  outline: none;
+  background-color: transparent;
+}
+
+//-----------------------Landing item-----------------------------
+
+.custom-button{
+  display: inline-block;
+  font-weight: 400;
+  text-align: center;
+  white-space: nowrap;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  background-color: white;
+  user-select: none;
+  border: 1px solid #8367D8;
+  padding: 12px 20px;
+  font-size: 1rem;
+  line-height: 1.5;
+  transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out, border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+}
+
+.item{
+  .custom-button{
+    min-width: 300px;
+  }
+}
+
+.categories{
+  display: block;
+  background-color: white;
+  margin-bottom: 0px;
+  padding: 0;
+  border-top: 1px solid $light-grey;
+
+  h4{
+    padding-top:10px;
+    padding-left: 20px;
+    color: blue;
+  }
+}
+
+.category-item{
+  display: block;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  border-bottom: 1px solid $light-grey;
+  
+  &:last-child{
+      border-bottom: 0;
+  }
+}
+.category-item-link{
+  padding-left: 20px;
+  font-size: 1.2rem;
+  color: black;
+  &:hover{
+    text-decoration: none;
+  }
+}
+.item{
+  margin-top: 50px;
+  margin-bottom: 50px;
+  padding-top: 70px;
+}
+
+//--------------------Feature-----------------------------
+
+#feature-collection .thumb{
+  height: 350px;
+  background-position: center center !important;
```

Comment:
```css
!important
```
treba izbjegavati ako je to ikako moguće. 
Potreba za `!important` najčešće ukazuje na porešno raspoređen DOM ili css.